include/coap2/option.h

```diff
@@ -88,34 +88,11 @@ size_t coap_opt_size(const coap_opt_t *opt);
 #error COAP_OPT_FILTER_SHORT + COAP_OPT_FILTER_LONG must be less or equal 16
 #endif /* (COAP_OPT_FILTER_SHORT + COAP_OPT_FILTER_LONG > 16) */
 
-/** The number of elements in coap_opt_filter_t. */
-#define COAP_OPT_FILTER_SIZE                                        \
-  (((COAP_OPT_FILTER_SHORT + 1) >> 1) + COAP_OPT_FILTER_LONG) +1
-
-/**
```

Comment:
I think that the comments that describe the meaning of the `coap_opt_filter_t` fields should be kept (probably not as doxygen comment but plain C comments instead.)

Reply:
Comment added and code pushed.